### PR TITLE
feat(Management): Add support for client credentials endpoints

### PR DESCRIPTION
**Pull request by evansims**
### Description

This PR adds support to the `Auth0\SDK\API\Management\Clients` class for managing client credentials using the Management API. The following methods were added:

- `Auth0\SDK\API\Management\Clients::createCredentials($clientId, $body)` issues a `POST` request to `/api/v2/clients/:client_id/credentials` with the supplied data.
- `Auth0\SDK\API\Management\Clients::getCredentials($clientId)` issues a `GET` request to `/api/v2/clients/:client_id/credentials` to retrieve a list of registered credentials for the specified client.
- `Auth0\SDK\API\Management\Clients::getCredential($clientId, $credentialId)` issues a `GET` request to `/api/v2/clients/:client_id/credentials/:credential_id` to retrieve the specified client credential.
- `Auth0\SDK\API\Management\Clients::deleteCredential($clientId, $credentialId)` issues a `DELETE` request to `/api/v2/clients/:client_id/credentials/credential_id` to delete the specified client credential.

The `ClientsTest` unit tests have been expanded to provide coverage for these new methods.

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### Checklist

- [x] My code follows the [contributing guidelines of this repo](./CONTRIBUTING.md)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass with my changes
- [x] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
- [x] I have read the [Auth0 code of conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

**Comment by evansims** [comment]
Please disregard the `Default Labels` workflow. Appears to be a bug I need to fix there, but it is unrelated to this PR.

**Comment by Widcket** [comment]
Are these private endpoints? These don't show up in https://auth0.com/docs/api/management/v2#!/Clients/get_clients

File: src/Contract/API/Management/ClientsInterface.php [hunk 1]
```diff
@@ -105,4 +105,67 @@ public function delete(
         string $id,
         ?RequestOptions $options = null,
     ): ResponseInterface;
+
+    /**
+     * Add new credentials to the specified Client.
+     *
+     * @param  string  $clientId Client to add credentials to
+     * @param  array<mixed>  $body  Additional body content to pass with the API request. See @see for supported options.
```
Comment (adamjmcgrath) [inline src/Contract/API/Management/ClientsInterface.php:113]: I'm assuming these will follow this convention https://github.com/auth0/auth0-python/blob/master/auth0/management/client_credentials.py#L77

These links will just have to the top of the mngmt api docs while we wait for the docs to be published. (or you can leave these links out and update them when this is GA'd)

Comment (evansims) [reply src/Contract/API/Management/ClientsInterface.php:113]: Added in https://github.com/auth0/auth0-PHP/pull/700/commits/2eedc725954f386f41e17007b0097451075a1771, thanks!

**Review by adamjmcgrath — APPROVED** [review]

**Pull request merged**